### ISSUE TITLE
Question about the data set up and loss

### ISSUE THREAD

**Issue by mavaylon1** [divamgupta/image-segmentation-keras#343]
Hi I saw in the tutorial that the image ground truths are integer class labels. Why then are we not using sparse cross entropy? Isn’t categorical cross entropy for one hot encoded masks?
Does the data generator one hot encode it from this format?

**Comment by divamgupta** [comment]
Yes, the generator is encoding them to one hot
